releasenotes/notes/add-examples-to-experiments-api-docs-31f3e3c3369a6f3d.yaml

```diff
@@ -1,32 +1,31 @@
 ---
 other:
   - |
-    Added minimal working code examples to the API pages for Experiments,
-    EFSpectroscopy,
-    EFRabi,
-    FineAmplitudeCal,
-    FineXAmplitudeCal,
-    FineSXAmplitudeCal,
-    FineDragCal,
-    FineXDragCal,
-    FineSXDragCal,
-    FineFrequencyCal,
-    FrequencyCal,
-    HalfAngleCal,
-    RoughAmplitudeCal,
-    RoughXSXAmplitudeCal,
-    EFRoughXSXAmplitudeCal,
-    RoughDragCal,
-    RoughFrequencyCal,
-    QuantumVolume,
-    InterleavedRB,
-    LayerFidelity,
-    StandardRB,
-    MitigatedProcessTomography,
-    MitigatedStateTomography,
-    ProcessTomography,
-    StateTomography and
-    TomographyExperiment.
-    By default, the backend used in the code examples, are simulators 
-    such as, SingleTransmonTestBackend() and
-    AerSimulator(FakePerth()).
+    Added minimal working code examples to the API pages for the experiments,
```

Comment:
When I built the docs locally, this note didn't appear. Maybe the problem was in my configuration. I will check the notes in the CI build when it is passing.

Reply:
This is the first line of the note.

Reply:
It shows up in the CI build, so we don't have to worry about it. I am not sure why local build didn't have the note.